Roll-ui fixes for the app

basit-nickelfox opened the pull request:
- Update the way SVG is used as it breaks the app
- also had to comment out **' export * from "./value" '**. This makes use of material-UI and uses HTML tags which breaks the app

marlon-wiprud reviewed (changes requested):
Looks like there are some strange whitespaces included from a linter you have configured?

Note that the way you have imported svg's breaks roll-ui storybook
![image](https://user-images.githubusercontent.com/37754165/172202145-4b55b474-f509-4e27-b048-8ecdd0e49751.png)

We must also consider how these changes will affect staking-ui
```
./src/organisms/stakeTokens/index.tsx
Attempted import error: 'Value' is not exported from 'roll-ui/src'.
```

basit-nickelfox commented:
> Looks like there are some strange whitespaces included from a linter you have configured?
> 
> Note that the way you have imported svg's breaks roll-ui storybook ![image](https://user-images.githubusercontent.com/37754165/172202145-4b55b474-f509-4e27-b048-8ecdd0e49751.png)
> 
> We must also consider how these changes will affect staking-ui
> 
> ```
> ./src/organisms/stakeTokens/index.tsx
> Attempted import error: 'Value' is not exported from 'roll-ui/src'.
> ```

updated and fixed
